src/coreclr/vm/arm64/asmhelpers.asm

```diff
@@ -676,6 +676,13 @@ COMToCLRDispatchHelper_RegSetup
 
 #endif ; FEATURE_HIJACK
 
+; void* PacStripPtr(void *);
+    LEAF_ENTRY PacStripPtr
+        ; xpaci    x0
+        DCD     0xDAC143E0
```

Comment:
interestingly, for windows, you don't have to check the availability of feature like we do on linux with `.arch_extension pauth`?

Reply:
The .arch_extension is primary to make clang emit the PAC instruction in binary. Here we directly put the binary equivalent of `xpaci x0` so compiler that doesn't support PAC yet, it won't complain about it. It doesn't cause harm on non-pac systems as the PAC instructions would fall into the nop space.